test_suite/slice.sugar.jsonnet

```diff
@@ -49,6 +49,14 @@ local arrCases = [
         output: arr,
     },
     {
+        input: arr[1::],
```

Comment:
Ah ok. Updated .fmt.golden. It looks like you made some changes but manually reverted anything that would have modified the tests so lgtm.
 https://github.com/google/jsonnet/commit/17c59acbfd455e42ab6bae0e5517157be4bf0b90#diff-5dd5fae04796372f66a075da6ff11ffc